setup.py

```diff
@@ -140,10 +118,9 @@ def get_blas_lib_dir(cls):
     "pecos.core.libpecos_float32",
     sources=["pecos/core/libpecos.cpp"],
     include_dirs=["pecos/core", "/usr/include/", "/usr/local/include"],
-    libraries=["gomp", "gcc"] + blas_lib,
-    library_dirs=blas_dir,
+    libraries=["gomp", "gcc", "stdc++"],
     extra_compile_args=["-fopenmp", "-O3", "-std=c++17"] + manual_compile_args,
-    extra_link_args=['-Wl,--no-as-needed', f"-Wl,-rpath,{':'.join(blas_dir)}"]
+    extra_link_args=['-Wl,--no-as-needed', f"-Wl,-rpath"]
```

Comment:
we might be able to remove the rpath path.

Reply:
Thanks for the suggestion. We completely remove the `extra_link_args` in the next Revision.